Gemfile

```diff
@@ -40,6 +40,6 @@ gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
 gem 'topological_inventory-api-client', '~> 1.0', :git => "https://github.com/ManageIQ/topological_inventory-api-client-ruby"
 
-gem 'approval-api-client-ruby', :git => 'https://github.com/ManageIQ/approval-api-client-ruby', :branch => "master"
+gem 'approval-api-client-ruby', :git => 'https://github.com/mkanoor/approval-client', :branch => "master"
```

Comment:
This was done to not update the ManageIQ gem which will break QE and Production

Reply:
@mkanoor is there a plan in place for when we can move this back to the ManageIQ version?

Reply:
@syncrou Once the whole solution has been implemented with the new tag based solution we will update the gem in ManageIQ